### PR TITLE
html file to contrastjs

### PR DESCRIPTION
**Pull request by ieliz**

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+<p id = "status">OpenCV.js is loading...</p>
+<div>
+  <div class = "inputoutput">
+    <img id = "imageSrc" src = "pic1.jpg"/>
+      <div class = "caption">
+        Run example: <input type = "button" id = "myButton" onclick = "update()" value = "Run" disabled = true />
+      </div>
+    <div class = "slide">
+    Contrast: <input type = "range" min = "1.0" max = "3.0" step = "0.1" value = "1.5" id = "Contrast" onchange = "update()" >
+    Brightness: <input type = "range" min = "0" max = "100" step = "1" value = "50" id = "Brightness" onchange = "update()" >
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html]: Please remove whitespaces for html attributes

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  </div>
+</div>
+
+<script type="text/javascript">
+
+//Source:
+//https://docs.opencv.org/master/d2/de8/group__core__array.html#ga3460e9c9f37b563ab9dd550c4d8c4e7d
+
+function update() {
+  let mat = cv.imread(document.getElementById('imageSrc'));
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html]:
```js
let mat = cv.imread('imageSrc');
```

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    <img id="imageSrc" src="pic1.jpg"/>
+      <div class="caption">
+        Run example:<input type="button" id="myButton" onclick="update()"
+        value="Run" disabled=true/>
+      </div>
+    <div class="slide">
+    Contrast:<input type="range" min="1.0" max="3.0" step="0.1" value="1.5"
+    id="Contrast" onchange="update()">
+    Brightness:<input type="range" min="0" max="100" step="1" value="50"
+    id="Brightness" onchange="update()">
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html]: Please use camelCase for `id` tags: "contrast" and "brightness".

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+<script type="text/javascript">
+
+//Source:
+//https://docs.opencv.org/master/d2/de8/group__core__array.html#ga3460e9c9f37b563ab9dd550c4d8c4e7d
+
+function update() {
+  let mat = cv.imread(document.getElementById('imageSrc'));
+  let dst = new cv.Mat();
+  let cont = Number(document.getElementById("Contrast").value);
+  let brig = Number(document.getElementById("Brightness").value);
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html]: Can we use these values directly without casting with `Number`?

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+  cv.imshow('canvasOutput', dst);
+  mat.delete();
+  dst.delete();
+};
+
+function onOpenCvReady() {
+  document.getElementById("status").innerHTML = "OpenCV.js is ready.";
+  cv["onRuntimeInitialized"] = () => {
+   document.getElementById("myButton").disabled = false;
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.html]: Please fix indentation: two whitespaces

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+
+
+<script type="text/javascript">
+
+function contrast() {
+  let mat = cv.imread("imageSrc"));
+  let dst = new cv.Mat();
+  let cont = Number("contrast".value);
+  let brig = Number("brightness".value);
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.md]: Is that correct?

Comment (ieliz) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.md]: What's here should be incorrect? I do not get it.

Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contrast.md]: So, it works or not? If it doesn't work - let's return to working version :)

**Pull request merged**